src/libstd/io/stdio.rs

```diff
@@ -204,6 +204,30 @@ impl Stdin {
     ///
     /// For detailed semantics of this method, see the documentation on
     /// `BufRead::read_line`.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
This should be tagged `no_run` as we probably don't want to run it during doctests.